ibis/expr/operations/analytic.py

```diff
@@ -28,7 +28,7 @@ def over(self, window):
 
     @property
     def inputs(self):
```

Comment:
IIRC this implementation of `inputs` is the primary reason for its existence.

Reply:
`inputs` is used by the pandas execution engine to filter out irrelevant arguments. I have a follow-up patch which removed this property entirely.